hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/RatisPipelineProvider.java

```diff
@@ -170,12 +169,18 @@ public Pipeline create(ReplicationFactor factor) throws IOException {
       throw new IllegalStateException("Unknown factor: " + factor.name());
     }
 
+    int nodeIdHash = 0;
+    if (factor == ReplicationFactor.THREE) {
```

Comment:
NIT: this condition check can be combined into the switch(factor) after line 165.

Reply:
Good catch. Updated.